sopel/module.py

```diff
@@ -248,8 +248,10 @@ def add_attribute(function):
     return add_attribute
 
 
-def rate(value):
-    """Decorate a function to limit how often a single user may trigger it.
+def rate(user_rate, channel_rate=0, global_rate=0):
```

Comment:
Any particular reason not to also specify a default value for `user_rate`? It's the first positional argument, so it should remain backward-compatible without requiring any particular limit to be specified in new modules.